frontend/clueless.js

```diff
@@ -1,6 +1,7 @@
 var tag = "";
 var isMyTurn = true;
-var isEvidenceSelectionTime = false;
+var canSuggest = true;
```

Comment:
Instead of all these booleans, we could have a state machine of enums that are like NOT_TURN, START_TURN, SUGGEST, END_TURN, etc. I don't care too much about it at this point though.